distributions/debian/copyright

```diff
@@ -1,10 +1,10 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: Jamulus
-Upstream-Contact: Volker Fischer 
-Source: https://github.com/corrados/jamulus/
+Upstream-Contact: The Jamulus Development Team
+Source: https://github.com/jamulussoftware/jamulus/
 
 Files: *
-Copyright: 2004-2020 Volker Fischer 
+Copyright: 2004-2022 The Jamulus Development Team
```

Comment:
Since AFAIK, Volker did it almost alone for the beginning, I'd suggest:
* Copyright: 2004-20XX Volker Fischer 
* Copyright: 20XX-2022 The Jamulus Development Team

(with replacements of the XX by relevant year)

Reply:
2019 was the first time anyone but Volker appears in the commit history -- but I'm not sure how accurate that is as he wasn't that au fait with git to start with and did some manual merge / fix / commit cycles that lost history.